src/Hierarchy/HierarchicalArrayCachePoolTrait.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+/*
+ * This file is part of php-cache organization.
+ *
+ * (c) 2015 Aaron Scherer <aequasi@gmail.com>, Tobias Nyholm <tobias.nyholm@gmail.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Cache\Hierarchy;
+
+/**
+ * @author Tobias Nyholm <tobias.nyholm@gmail.com>
+ */
+trait HierarchicalArrayCachePoolTrait
```

Comment:
This trait is only used once. Lets not have a trait and instead put in in the ArrachCachePool class.

Reply:
Was planning to use it for the nested directories for Filesystem Adapter ;)

Reply:
@Nyholm any updates ?

Reply:
It is just one function and this trait is using another trait. 

Im hesitant about this. Understanding the hierarchy cache is difficult enough. By removing this trait and possible duplicate the 3 lines of code it's providing will make things easier to read and understand